doc/nrf/known_issues.rst

```diff
@@ -1650,7 +1677,7 @@ NCSDK-8372: Project name collision causes SES Nordic Edition to load the wrong p
   **Workaround:** If the path in :guilabel:`Build Directory` points to the wrong project, select the correct project by using the :guilabel:`...` button for :guilabel:`Projects` and navigating to the correct project location.
   The build directory will update automatically.
 
-.. rst-class:: v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-1 v1-5-0
+.. rst-class:: v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-1 v1-5-0
```

Comment:
This is fixed in SES-NE v5.68 which will be included for NCS v1.8.0

Reply:
I might have marked the wrong line.

What is fixed is this one: `NCSDK-8372: Project name collision causes SES Nordic Edition to load the wrong project`